src/command/check/check.c

```diff
@@ -30,9 +30,9 @@ checkManifest(void)
     MEM_CONTEXT_TEMP_BEGIN()
     {
         // Loop through all defined databases and attempt to build a manifest
-        for (unsigned int pgIdx = 0; pgIdx < cfgOptionIndexTotal(cfgOptPgPath); pgIdx++)
+        for (unsigned int pgIdx = 0; pgIdx < cfgOptionGroupIndexTotal(cfgOptGrpPg); pgIdx++)
         {
-            if (cfgOptionTest(cfgOptPgHost + pgIdx) || cfgOptionTest(cfgOptPgPath + pgIdx))
+            if (cfgOptionGroupIndexTest(cfgOptGrpPg, pgIdx))
```

Comment:
But `cfgOptionGroupIndexTest(cfgOptGrpPg, pgIdx)` indicates something in the PG group was set but that doesn't mean the pgHost or pgPath is set, so shouldn't we still be checking that?

Reply:
According to the rules in Data.pm *all* pg options require either pg-path or pg-host to be set before they can be set (see CFGDEF_DEPEND at line 2604 for an example). So one of those two options must be set if any are set.